gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
```

Comment:
Is it common to have to gradle wrappers in the repository? A second one was added in the buildSrc directorey.

Reply:
has been removed